atlasexec/atlas_migrate.go

```diff
@@ -383,6 +402,58 @@ func (c *Client) MigrateStatus(ctx context.Context, params *MigrateStatusParams)
 	return firstResult(jsonDecode[MigrateStatus](c.runCommand(ctx, args)))
 }
 
+// MigrateDiff runs the 'migrate diff' command and returns the generated migration files.
```

Comment:
Mention the --dry-run in the GoDoc. It's a special type of migrate diff, it's "migrate diff --dry-run".